src/connection/Form.ts

```diff
@@ -90,15 +90,11 @@ export default class Form {
       }
 
       this.items.forEach((fields, itemIndex) => {
-         if (fields.length !== this.reportedFields.length) {
-            throw new InvalidParameterError(`Item ${itemIndex} does not contain all "reported" fields.`);
-         }
-
          this.reportedFields.forEach((field, index) => {
```

Comment:
Because you mentioned in an earlier version: "In which case is the server returning more fields than reported fields?"
And i think we could trust the servers implementation because we started with the extended form, too.

Reply:
We can't trust the server, because it's not always "our" server. If there is no issue with the check, what is the point in removing it?